### ISSUE TITLE
Test clients against the nexus / reva

### ISSUE THREAD

**Issue by butonic** [owncloud/client#7232]
To visualize what works and what does not, I would like to get automatic testing against [nexus](https://github.com/owncloud/nexus) / reva. Nightly runs would be awesome, but an initial test is welcome as well. I am still working on the sharing API, but file sync should work. But to what extent?

@michaelstingl mentioned @HanaGemela might have a look?

**Comment by michaelstingl** [comment]
Maybe perform the Smoke Test against the nexus / reva setup. (Lower priority than 2.6.0 release of course)

@butonic Any hint to how to fire up the nexus / reva? Link to HowTo?

**Comment by guruz** [comment]
Is this something for @mrow4a and smashbox?

**Comment by michaelstingl** [comment]
> Is this something for @mrow4a and smashbox?

Oh, yeah – this would be awesome. @mrow4a @patrickjahns we need a smashbox intro for @HanaGemela @jnweiger @TheOneRing

**Comment by mrow4a** [comment]
@michaelstingl @guruz yes, probably can give few commands to run and point to Quickstart guide for smashbox (I will update it too). 

What client branch is involved? Will it be a new test or standard sharing test?

**Comment by mrow4a** [comment]
To quickly start this can be helpful https://github.com/owncloud/smashbox/tree/update_readme#quickstart 

On how to write a test this is best
https://github.com/owncloud/smashbox/blob/update_readme/lib/examples/test_hello.py

**Comment by michaelstingl** [comment]
@mrow4a Client not modified. Nexus/Reva tries to be fully backwards compatible to oC10.
